src/patterns/components/tables/react/code/secondary-row-comparison.hbs

```diff
@@ -0,0 +1,13 @@
+  <SprkTable 
+    variant="secondaryRowComparison"
+    additionalTableClasses="sprk-b-Table--spacing-medium"
```

Comment:
If there is only one additionalClasses for the table, this prop can be shortened to `additionalClasses`

Reply:
There are additionalClasses and additionalTableClasses. additionalClasses are applied to the div wrapping the table element. additionalTableClasses are applied to the table element. I can certainly shorten both though. addClasses and addTableClasses ?

Reply:
I see now, there's the tablecontainer and table. Could indicate that ones the container. We've been meaning to talk about add vs. additional for naming.

Reply:
Yeah. In my actual component I use the additionalWrapperClasses variable. I can update it throughout for clarity. If we move to "add" instead of "additional" I can go back and update.